src/Captioning/Format/SubripFile.php

```diff
@@ -116,4 +155,14 @@ private function validateOptions(array $options)
         }
         return true;
     }
+
+    /**
+     * @param \DateTime $startTimeline
+     * @param \DateTime $endTimeline
+     * @return boolean
+     */
+    private function validateTimelines(\DateTime $startTimeline, \DateTime $endTimeline)
+    {
+        return !($startTimeline >= $endTimeline);
```

Comment:
or just simply (the negation isn't necessary):
```php
return ($startTimeline < $endTimeline);
```

But considering the fact that milliseconds can't be compared with DateTimes, this method will certainly change.